pom.xml

```diff
@@ -4,7 +4,7 @@
     <groupId>com.griefcraft.lwc</groupId>
     <artifactId>Modern-LWC</artifactId>
     <name>Modern-LWC</name>
-    <version>2.1.1</version>
+    <version>2.1.2</version>
```

Comment:
@lifehome Don't change pom.xml please. Also, you should target the dev branch, because master is currently only for stable releases.

Aside from that, it looks OK. Can you let me know how you have it set up (and possibly send some sample econ files) so that I may do some testing before merging? I believe we have each other on Discord so you can message me there if you like.

Thanks